### DISCUSSION TITLE
asdf plugin?

### DISCUSSION THREAD

**Discussion by lemeurherve** [updatecli/updatecli#567] [Ideas]
Allow updatecli to be installed via [asdf](https://github.com/asdf-vm/asdf) would be nice and can help gain exposure

**Comment by olblak** [comment]
that's a great idea

**Comment by treezio** [comment]
I created yesterday https://github.com/treezio/asdf-updatecli , sorry I did not see this discussion.

Anyway, I could transfer this to `updatecli` organization 👍🏻 

P.S. Currently waiting to be integrated in `asdf-plugins` public repository https://github.com/asdf-vm/asdf-plugins/pull/907

**Reply by olblak** [reply]
This is really awesome.

I think this would be better to have this repository in the Updatecli organization as other Updatecli maintainers would be able to help
And I would be super happy to invite you in the org to maintain that repository :D

I think it would be nice to also update installation instruction to mention asdf installation instruction 
-> https://github.com/updatecli/website/blob/master/content/en/docs/prologue/installation.adoc

And if need, also updating the Updatecli manifest which automate installation instruction update 
-> https://github.com/updatecli/website/blob/99104678e80653136a6c86bd1cfed3a974e97481/updatecli/updatecli.d/installation.yaml#L39

**Reply by treezio** [reply]
Yeah, sure, why not?

Thinking about the timeline. I guess the best option is to:
- [x] 1. Transfer the repository to the`updatecli` organization
- [x] 2. Refactor/Recreate the Pull Request to the `asdf-plugins` repository
- [x] 3. Update installation Instructions (it is different using the `asdf-plugins` repository or the "external" repository)

What do you think @olblak ?

**Reply by olblak** [reply]
This would be great, I just sent you an organization invite so you should be able to transfert the repository

**Reply by treezio** [reply]
Repository transferred ✅

**Reply by treezio** [reply]
Pull Request information updated https://github.com/asdf-vm/asdf-plugins/pull/907 ✅

**Reply by dduportal** [reply]
Woooo thanks @treezio that is aweome !
